src/vm/functions/mod.rs

```diff
@@ -88,6 +88,8 @@ define_versioned_named_enum!(NativeFunctions(ClarityVersion) {
     BuffToIntBe("buff-to-int-be", ClarityVersion::Clarity2),
     BuffToUIntBe("buff-to-uint-be", ClarityVersion::Clarity2),
     IsStandard("is-standard", ClarityVersion::Clarity2),
+    ParsePrincipal("parse-principal", ClarityVersion::Clarity2),
+    AssemblePrincipal("assemble-principal", ClarityVersion::Clarity2),
```

Comment:
What's your take on `principal-new` instead of `principal-assemble`. this function looks like a constructor to me.

Reply:
Or `principal-construct` and `principal-destruct` ?

Reply:
I like `principal-construct` & `principal-parse`

Reply:
I don't have strong feelings. Who wants to make the call?

Reply:
Pick something you like, as long as it works with the convention. We will do a pass before 2.1, to make sure that there is some naming consistency with all the the functions that we introduced :)